package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "webpack-boilerplate",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Please add `start` and `watch` scripts for starting the project with webpack